### PR TITLE
ACL2024 Workshop Ingestion

### PR DESCRIPTION
**Pull request by anthology-assist**

### CODE REVIEW

File: data/yaml/venues/knowledgenlp.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+acronym: KnowledgeNLP
```
Comment (mjpost) [inline data/yaml/venues/knowledgenlp.yaml:1]: If this is the third, can you make sure that we don't have this venue already?

File: data/yaml/venues/knowledgenlp.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+name: The 3rd Workshop on Knowledge Augmented Methods for NLP
```
Comment (mjpost) [inline data/yaml/venues/knowledgenlp.yaml:2]: The venue name should not include "The 3rd". That is a particular meeting of the venue, not the venue name itself.

File: data/yaml/venues/knowledgenlp.yaml (hunk 1 above)
Comment (mjpost) [reply data/yaml/venues/knowledgenlp.yaml:1]: Here is their website: https://knowledge-nlp.github.io/acl2024/

It appears they were not previously associated with ACL conferences.

Comment (anthology-assist) [reply data/yaml/venues/knowledgenlp.yaml:1]: @mjpost Yes I also checked. So I'm assuming it's okay to have this workshop added as a new venue?

File: data/yaml/venues/knowledgenlp.yaml (hunk 2 above)
Comment (anthology-assist) [reply data/yaml/venues/knowledgenlp.yaml]: Addressed.

File: data/yaml/venues/knowledgenlp.yaml (hunk 1 above)
Comment (mjpost) [reply data/yaml/venues/knowledgenlp.yaml:1]: Yes it seems to check out.

File: data/yaml/venues/ml4al.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+acronym: ML4AL
+name: 1st Workshop on Machine Learning for Ancient Languages
```
Comment (mjpost) [inline data/yaml/venues/ml4al.yaml:2]: ~~1st~~ Workshop on Machine Learning for Ancient Languages

File: data/yaml/venues/teachnlp.yaml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+acronym: TeachNLP
+name: The Sixth Workshop on Teaching NLP
```
Comment (mjpost) [inline data/yaml/venues/teachnlp.yaml:2]: This is the sixth meeting, so it seems like it probably exists? Digging around I found this: https://aclanthology.org/venues/teachingnlp/

File: data/xml/2024.teachnlp.xml
Comment (mjpost) [inline data/xml/2024.teachnlp.xml:1]: This should be reingested using [the existing venue ID](https://aclanthology.org/venues/teachingnlp/)

File: data/yaml/venues/ml4al.yaml (hunk 3 above)
Comment (anthology-assist) [reply data/yaml/venues/ml4al.yaml]: Done.

File: data/xml/2024.teachnlp.xml
Comment (anthology-assist) [reply data/xml/2024.teachnlp.xml:1]: Done

**Comment by anthology-assist** [comment]
@mjpost aclpub2 doesn't include `sig` info, not sure where we can get it...?

**Review by mjpost — APPROVED** [review]

**Pull request merged**